src/EcologyClub.scala

```diff
@@ -0,0 +1,26 @@
+// This class provides general information about the number of members in the Ecology Club, the meeting time and day
+
+case class EcologyClub (numOfMembers: Int, meetingDay: String, meetingTime: Int){
+def clubName() = println("Welcome to the Ecology Club!")
+}
+
+object Club {
+
+
+  def main(args: Array[String]): Unit = {
+    val club1 =  EcologyClub(10, "Monday", 1600)
+    var club2 =  ArtClub(11, "Tuesday", 1700)
+    var club3 =  DanceClub(30, "Friday", 1700)
+
+    club1.clubName()
+    println(s"Come Grow with us at ${club1.meetingTime} on ${club1.meetingDay} ")
```

Comment:
This is fine but would be more practical to use `override val toString` and put both your clubName() and this formatted string in it. Then you'd only have one line in your main() for each class like this:   
 println( club1 )
etc.